test/Conversion/TritonToStructured/addptr_reshape_broadcast.mlir

```diff
@@ -36,7 +36,7 @@ module {
 // CHECK:         tt.func @kernel([[PARAM_0_:%.+]]: !tt.ptr<bf16, 1>, [[PARAM_1_:%.+]]: !tt.ptr<bf16, 1>) {
 // CHECK-DAG:       [[CST_6144_:%.+]] = arith.constant 6144 : index
 // CHECK-DAG:       [[CST_6_:%.+]] = arith.constant 6 : index
-// CHECK:           [[VAR_0_:%.+]] = tts.make_tptr [[PARAM_1_]] to sizes: [256, 128], strides: [1, [[CST_6_]]{{.}}, offsets: [512, [[CST_6144_]]{{.}}, parent_sizes: [0, 0] : <bf16, 1> to tensor<256x128x!tt.ptr<bf16, 1>>
+// CHECK:           [[VAR_0_:%.+]] = tts.make_tptr [[PARAM_1_]] to sizes: [256, 128], strides: [1, [[CST_6_]]{{.}}, offsets: [512, [[CST_6_]]144], shape: [0, 0], order: [] : <bf16, 1> to tensor<256x128x!tt.ptr<bf16, 1>>
```

Comment:
there's a typo here i think, should be `[[CST_6144_]]` instead of `[[CST_6_]]144]`